### PR TITLE
Adding useless functionality to our software

### PR DESCRIPTION
**Pull request by nolaspring**
I love my mama!!

### CODE REVIEW

**Comment by cmendez1701** [comment]
this is a terrible idea. youre a loser

File: nolaspring_easteregg [hunk 1]
```diff
@@ -0,0 +1 @@
+When you hit F5 will open up mario brothers in a new window
```
Comment (nolaspring) [inline nolaspring_easteregg]: I think this is amazing, so does QA

**Comment by franks999** [comment]
Hey @nolaspring keep that useless functionality coming!

**Comment by adilinusah** [comment]
This sucks !!!! What are doing

File: nolaspring_easteregg (hunk 1 above)
Comment (nolaspring) [inline nolaspring_easteregg]: Added this to get people to use our software

**Comment by nolaspring** [comment]
please check to see if this is any better @adilinusah @cmendez1701 @franks999

**Pull request merged**